### PR TITLE
HCAP-882 | Withdraw design updates

### PR DESCRIPTION
**Pull request by arranfw**
This is my attempt at deciphering the different notes in the ticket

- New Withdraw button text
- New copy on the confirmation dialogue
- New styling + copy on success page

image
image
image

### CODE REVIEW

**Review by fwkendall — APPROVED** [review]

**Review by dbayly-freshworks — APPROVED** [review]
Looks ok to me!

File: client/src/pages/private/ParticipantEOI.js [hunk 1]
```diff
@@ -296,7 +297,7 @@ export default () => {
                       color='primary'
                       onClick={() => setOpenWithdraw(true)}
                     >
-                      Withdraw From the Program
+                      Withdraw PEOI
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantEOI.js:300]: Don't you think PEOI is a bit technical term. What about `Withdraw Interest`?

**Review by fwpushan — APPROVED** [review]

File: client/src/pages/private/ParticipantEOI.js (hunk 1 above)
Comment (arranfw) [reply client/src/pages/private/ParticipantEOI.js:300]: Fair comment, I just copied the ticket, want to bring this up to the channel?

Comment (fwpushan) [reply client/src/pages/private/ParticipantEOI.js:300]: Yes, do it.

**Pull request merged**